File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portfliodn.java
// EXPECTED: 66
// Reference emit for PORTFLIODN — PORTFLIO COPY-linked type+status RC sum.
public final class PortfliodnRef {
  static int typeRc(String t) {
    if ("I".equals(t)) return 10;
    if ("C".equals(t)) return 20;
    if ("T".equals(t)) return 30;
    return 99;
  }

  static int statusRc(String s) {
    if ("A".equals(s)) return 1;
    if ("C".equals(s)) return 2;
    if ("S".equals(s)) return 3;
    return 99;
  }

  public static void main(String[] args) {
    int total = 0;
    for (String t : new String[] {"I", "C", "T"}) total += typeRc(t);
    for (String s : new String[] {"A", "C", "S"}) total += statusRc(s);
    System.out.println(total);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portfliodn.cs
// EXPECTED: 66
// Reference emit for PORTFLIODN — PORTFLIO COPY-linked type+status RC sum.
using System;
public static class PortfliodnRef {
  static int TypeRc(string t) {
    if (t == "I") return 10;
    if (t == "C") return 20;
    if (t == "T") return 30;
    return 99;
  }

  static int StatusRc(string s) {
    if (s == "A") return 1;
    if (s == "C") return 2;
    if (s == "S") return 3;
    return 99;
  }

  public static void Main() {
    int total = 0;
    foreach (var t in new[] { "I", "C", "T" }) total += TypeRc(t);
    foreach (var s in new[] { "A", "C", "S" }) total += StatusRc(s);
    Console.WriteLine(total);
  }
}
